### DISCUSSION TITLE
does not connect DB on arm64

### DISCUSSION THREAD

**Discussion by kikimora12304855** [kheeklab/privacyidea-docker#85] [Q&A]
does not want to connect DB on arm64, outputs error 500 on sqlite and postgresql, I am new to this and do not know what to do ( I do not know English I use google translator )

**Comment by mr-ssd** [comment]
@kikimora12304855 thank you for reporting the issue.

Could you please provide the details of the error such as configuration, error message, error log, etc.?

I don't test much on arm64 due to a lack of hardware to test. You should try to test on amd64 first to see if you getting the same error.

**Comment by kikimora12304855** [comment]
unfortunately I don't have any other hardware for the test I only have arm64

**Comment by mr-ssd** [comment]
@kikimora12304855 that's OK but if you could provide me the error log or screenshot of the error?

**Comment by kikimora12304855** [comment]
I followed the instructions. 
```shell
make build
``` 
```shell
make run
``` 

https://github.com/Khalibre/privacyidea-docker/assets/57181966/028fd325-ec84-41cc-9e42-5f8034a316a4

I apologize for the long replies, I don't always have internet.

**Comment by mr-ssd** [comment]
@kikimora12304855 in sqlresovler could you try `mysql+pymysql` I think it would work.

![image](https://github.com/Khalibre/privacyidea-docker/assets/9067408/22a459e9-b620-46ff-b4e9-d730754c4901)

**Comment by kikimora12304855** [comment]
<img width="1440" alt="изображение" src="https://github.com/Khalibre/privacyidea-docker/assets/57181966/d68f0e00-3841-48be-bb0a-0d334e4f3ac5">

**Comment by mr-ssd** [comment]
@kikimora12304855 could you please make the full screenshot of the configuration? Look like you enter wrong database address or port.

**Comment by kikimora12304855** [comment]
I did everything according to the instructions, i.e. make build, make run.
I haven't changed or touched docker-compose.

**Comment by mr-ssd** [comment]
@kikimora12304855 I mean you might enter the IP address incorrectly.
![image](https://github.com/Khalibre/privacyidea-docker/assets/9067408/eda49350-eb49-44d9-831e-7df77a4e06c3)

IP address here should be the MySQL server IP address not 127.0.0.1.

**Comment by kikimora12304855** [comment]
Thank you for your help, after you told me this I realized that I did not have BD installed, I tried to install mysql but got an error that this device can not be installed, I tried to install PostgreSQL and it worked, I found the ip address of BD but I did not understand what driver to use. And finally I understood how to copy logs.

**Reply by mr-ssd** [reply]
I'm glad to hear that helps. For PostgreSQL please use `postgresql+psycopg2` as driver.
